src/main/resources/static/codegen/built-api-spec.yml

```diff
@@ -3202,6 +3202,13 @@ paths:
           schema:
             type: string
             format: uuid
+        - name: cruManagementAreaId
```

Comment:
could you please add

`deprecated: true` to the apAreaId parameter above, and add a note in the description saying cruManagementAreaId should be used instead?

Reply:
Done this.